pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line-length = 88
+
+[tool.coverage.paths]
+source = ["singer", "*/site-packages"]
```

Comment:
Why include `*/site-packages` in coverage paths?

Reply:
This is so combined coverage is reported if the library is _installed_ by a tool like nox or tox and for example `.tox/*/lib/python*/site-packages/singer` is seen as equivalent to the root `singer` package.

So you're right it's not needed here until such a tool is used (or MacOS and Windows are added to the test matrix, I think) :)